RelayProofsOfConcept/EddsaTokioServer/examples/eddsa_key_gen_client.rs

```diff
@@ -431,23 +345,40 @@ impl<T: Peer> Client<T> {
             protocol_id,
             last_message: RefCell::new(ClientMessage::new()),
             bc_dests: (1..(capacity + 1)).collect(),
-            timeout: 100, // 3 second delay in sending messages
+            timeout: 0, // 3 second delay in sending messages
```

Comment:
cool, are you sure no problems are caused because the lack of delay? 
- you can remove the comment as well 
- if you are not using delay consider removing it from the code altogether